### PR TITLE
KP-23ZBS-ACE from Climax Technology, a keypad

### PR DESCRIPTION
**Pull request by OsteHovel**
Adding support for KP-23ZBS-ACE
This is a keypad from Climax Technology

### CODE REVIEW

File: devices/climax.js [hunk 1]
```diff
@@ -111,4 +111,15 @@ module.exports = [
         toZigbee: [],
         exposes: [e.carbon_monoxide(), e.battery_low(), e.tamper(), e.battery()],
     },
+    {
+        zigbeeModel: ['KP-ACE_00.00.03.12TC'],
+        model: 'KP-23ZBS-ACE',
+        vendor: 'Climax Technology',
+        description: 'Zigbee Remote Keypad',
+        meta: {commandArmIncludeTransaction: true},
```
Comment (Koenkk) [inline devices/climax.js:119]: `commandArmIncludeTransaction` is not a valid option here, why did you add it?

**Comment by OsteHovel** [comment]
Hold on, it seems I copied the content from the wrong file...
